### DISCUSSION TITLE
In Next.js v13, I can't push code using command. I used push function of vscode editor to push code.

### DISCUSSION THREAD

**Discussion by cupid20103** [community/community#57981] [Programming Help]
### Body

Hello, Everyone!
I have some issues pushing my code.
This is Husky's pre-push command that runs before pushing my code.

```
#!/usr/bin/env sh
. "$(dirname -- "$0")/_/husky.sh"

yarn build
```

but I get same error like this.

```
stdin is not a tty
husky - pre-push hook exited with code 1 (error)
error: failed to push some refs to 'https://github.com/Gods-of-love/learn-nextjs-v13.git'
```

How to solve it?
This is my [github repo ](https://github.com/Gods-of-love/learn-nextjs-v13.git).

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Beargruug** [comment]
I think it's related to https://github.com/yarnpkg/yarn/issues/2998

Here's a workaround, create .husky/common.sh:
```
command_exists () {
  command -v "$1" >/dev/null 2>&1
}

# Git Bash and Yarn workaround (Windows)
if command_exists winpty && test -t 1; then
  exec < /dev/tty
fi
```
Source it in commit-msg, pre-commit and pre-push:

```
#!/bin/sh
. "$(dirname "$0")/_/husky.sh"
. "$(dirname "$0")/common.sh"

yarn ...
```
It doesn't seem to be necessary in places where just yarn install is executed.

I hope this will help you!
